### ISSUE TITLE
about metric

### ISSUE THREAD

**Issue by wdkang4715** [SwinTransformer/MIM-Depth-Estimation#11]
I trained my own dataset. But I got the below message.

![image](https://github.com/SwinTransformer/MIM-Depth-Estimation/assets/148187933/3bad912e-d69c-42ce-b4ce-59b5525e6bd5)

So I check the image size and ground-truth value.
when I check the iamge size 3011X4008 and the ground truth min : 0.0 the ground turth max : about 730.0

and my command line is like below:

 python3 train.py --dataset my_data --max_depth 730.0 --max_depth_eval 730.0 --data_path ../data/  --backbone swin_large_v2 --depths 2 2 18 2 --num_filters 32 32 32 --deconv_kernels 2 2 2 --window_size 30 30 30 15 --pretrain_window_size 12 12 12 6 --use_shift True True False False --flip_test --shift_window_test --shift_size 2 --pretrained weights/swin_v2_large_simmim.pth --save_model --crop_h 480 --crop_w 480 --layer_decay 0.85 --drop_path_rate 0.5 --log_dir logs/ --save_result
